### PR TITLE
Ignore auth headers in AjaxDependencyPlugin

### PR DESCRIPTION
**Pull request by Karlie-777**

### CODE REVIEW

File: shared/AppInsightsCommon/src/Interfaces/ICorrelationConfig.ts [hunk 1]
```diff
@@ -39,5 +39,10 @@ export interface ICorrelationConfig {
      */
     ajaxPerfLookupDelay?: number;
 
-    correlationHeaderDomains?: string[]
+    correlationHeaderDomains?: string[];
+
+    /**
+     * Response and request headers to be excluded from ajax tracking data.
+     */
+    ignoreHeaders?: string[];
```
Comment (MSNev) [inline shared/AppInsightsCommon/src/Interfaces/ICorrelationConfig.ts:47]: Looks good, can you also add this to the different documentation places (*.md) any of them that already have a reference to the "enableRequestHeaderTracking" and the response ones.

And include the currently known defaults to the documentation in the *,md and on this item.

Comment (MSNev) [reply shared/AppInsightsCommon/src/Interfaces/ICorrelationConfig.ts:47]: The documentation should also mention that if they want to include/override the default values they will need to add this to the config as an empty array (as the getConfig() helper function will ignore null and undefined values)

Comment (MSNev) [reply shared/AppInsightsCommon/src/Interfaces/ICorrelationConfig.ts:47]: Here is a list of some headers that we should exclude
https://www.loginradius.com/blog/async/everything-you-want-to-know-about-authorization-headers/

Comment (MSNev) [reply shared/AppInsightsCommon/src/Interfaces/ICorrelationConfig.ts:47]: Credentials is a request header that looks like it's used by ASP.Net and WWW-Authenticate is a response header

Comment (Karlie-777) [reply shared/AppInsightsCommon/src/Interfaces/ICorrelationConfig.ts:47]: it seems that `Authorization: <type> Credential=......`  Credentials might be a parameter of Authorization, so I didn't include Credentials. All header class in ASP.Net https://docs.microsoft.com/en-us/dotnet/api/microsoft.net.http.headers.headernames?view=aspnetcore-5.0#fields

**Review by MSNev — APPROVED** [review]

**Pull request merged**